Explain how the S3 connector uses AWS configuration files (#3856)
tenzir/tenzir: web/docs/connectors/s3.md

[default credentials provider
chain](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials-chain.html).

:::info
Make sure to configure AWS credentials for the same user account that runs
`tenzir` and `tenzir-node`. The AWS CLI creates configuration files for the
current user under `~/.aws`, which can only be read by the same user account.

The `tenzir-node` systemd unit by default creates a `tenzir` user and runs as

> that user, meaning that it the AWS credentials must also be configured for that

Reviewer: ```suggestion
that user, meaning that the AWS credentials must also be configured for that
```

> user account. `~/.aws` must be readable for the `tenzir` user in this setup.

Reviewer: ```suggestion
user. The directory `~/.aws` must be readable for the `tenzir` user.
```
